modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -22,3 +22,19 @@ function oe_content_publication_post_update_00001_update_field_labels(array &$sa
     $field_config->save();
   }
 }
+
+/**
+ * Make RDF fields not translatable.
+ */
+function oe_content_publication_post_update_00002_not_translatable_rdf_field(array &$sandbox): void {
```

Comment:
We cannot change existing sites' translatability. So we need to move this update path to EWCMS.

Reply:
done here https://github.com/ec-europa/ewcms/pull/197